components/fileselect/overview.md

```diff
@@ -69,6 +69,10 @@ Additionally, you may define an external drop zone by using the [Telerik UI for
 
 The FileSelect includes [built-in client-side validation]({%slug fileselect-validation%}) for the file size and type (extension). Additional custom validation can take place in the [OnSelect event]({%slug fileselect-events%}#onselect).
 
+## Initial Files
+
+The FileSelect `Files` property is a collection of type `IEnumerable<FileSelectFileInfo>` that holds a set of pre-selected files. These files will be initially displayed in the FileSelect files list. This functionality is helpful when you want to show files that were previously uploaded. [Read more about the Telerik FileSelect Initial Files feature...]({%slug fileselect-initial-files%})
```

Comment:
The technical details must be in the article dedicated to the described feature.
```suggestion
The Initial Files feature allows you to display a set of pre-selected files in the FileSelect when the component loads. This functionality is helpful when you want to show files that were previously uploaded. [Read more about the FileSelect Initial Files feature...]({%slug fileselect-initial-files%})
```